internal/format/templates/pretty.tmpl

```diff
@@ -30,19 +30,24 @@
         {{- range . }}
             {{- printf "module.%s" .Name | colorize "\033[36m" }}{{ printf " (%s)" .FullName }}
         {{ end -}}
-        {{- printf "\n\n" -}}
     {{ end -}}
+    {{- printf "\n\n" -}}
 {{ end -}}
 
-{{- if .Settings.ShowResources -}}
+{{- if or .Settings.ShowResources .Settings.ShowDataSources -}}
     {{- with .Module.Resources }}
         {{- range . }}
             {{- $url := ternary .URL (printf " (%s)" .URL) "" }}
-            {{- printf "resource.%s (%s)" .Spec .GetMode | colorize "\033[36m" }}{{ $url }}
-        {{ end -}}
-    {{ end -}}
+            {{- if $.Settings.ShowResources -}}{{- if eq "resource" (printf "%s" .GetMode) }}
+                {{- printf "resource.%s (%s)" .Spec .GetMode | colorize "\033[36m" }}{{ $url }}
+            {{ end -}}{{ end -}}
+            {{- if $.Settings.ShowDataSources }}{{ if eq "data source" (printf "%s" .GetMode) }}
+                {{- printf "data.%s (%s)" .Spec .GetMode | colorize "\033[36m" }}{{ $url }}
+            {{ end -}}{{ end -}}
```

Comment:
To make it consistence with others, define both `$isResource` and `$isDataResource` but using two if blocks is fine here.